components/arm/xarm/xarm6_kinematics.json

```diff
@@ -61,17 +61,17 @@
                 "y": 190,
                 "z": 250,
                 "translation": {
-                    "x": 0,
+                    "x": 49.497474683058328537299531,
                     "y": 0,
-                    "z": 70
+                    "z": -49.497474683058328537299531
                 },
                 "orientation": {
-                    "type": "ov_degrees",
+                    "type": "quaternion",
```

Comment:
[q] What is the rationale behind the type swap?

Reply:
`NewPose` does `q.Real = o.Quaternion()` where `o` is the given orientation.
I just kept it as a quaternion because that is what the backend uses.
I am not strongly attached to having the type stay at `quaternion` and I can change it to `ov_degrees` :)

Reply:
Honestly, accurate is more important than representation to me, but since we are trying to be consistent with OV_Degrees, that is the preference.